_episodes/00-quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Quickstart Guide"
+teaching: 5
+exercises: 10
+questions:
+
+- "How do I activate the ESMValTool environment?"
+- "How do I configure the ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Activate the ESMValTool environment"
+- "Configure the ESMValTool"
+- "Successfully run a recipe"
+
+keypoints:
+- "See the installation guide for activating the ESMValtool environment."
+- "The config file has to be set up, uncommenting the environment you use."
+- "The command for running a recipe is `esmvaltool run`."
+---
+
+> ## Installing and getting started with the ESMValTool
+>
+> To install the ESMValTool please see 
+> [here](https://docs.esmvaltool.org/en/latest/quickstart/installation.html#)
+{: .callout}
```

Comment:
The main idea in issue https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/245 is that esmvaltool is already installed. Can we replace this with a note something like "we assume that you are working on a compute clusters where esmvaltool is installed and you have access to data. with a link to Setup https://esmvalgroup.github.io/ESMValTool_Tutorial/setup.html